### PR TITLE
leaper: allow comment override also on openSUSE

### PR DESCRIPTION
**Pull request by lnussel**

### CODE REVIEW

File: leaper.py [hunk 1]
```diff
@@ -404,6 +404,13 @@ def check_source_submission(self, src_project, src_package, src_rev, target_proj
                 self.needs_legal_review = True
                 return True
 
+        if origin_same is False and self.override_allow:
```
Comment (jberry-suse) [inline leaper.py]: Based on where `origin_same` is set this could be called when it is not set.

**Comment by jberry-suse** [comment]
Currently, one should be able to re-open request after making comment to achieve the same thing.

**Comment by lnussel** [comment]
yeah one could but this risks that things slip through

File: leaper.py (hunk 1 above)
Comment (lnussel) [reply leaper.py]: ? you lost me here

Comment (jberry-suse) [reply leaper.py]: If the package is not in lookup file (presumably new package) `origin_same` will not be set and this condition will cause crash.

```python
if package in self.lookup_150:
    origin = self.lookup_150[package]

# ...

if origin:
    origin_same = self._check_same_origin(origin, src_project)
    # ...
else:
    # ...., but not always return

# ...

# origin_same amay not be set if not origin
if origin_same is False and self.override_allow:
```

Unless there is some larger condition combination that is not obvious when looking at the code that would prevent this from happening, but given how long this function is that should likely be protected against in-case of future change even if it exists.

**Pull request merged**